telemetry_package/main.py

```diff
@@ -1,31 +1,140 @@
-import sys
+'''
+This code gets too many messages all at once. Hence it needs to: 
+    - The Node only sends the first message it receives
+'''
+
+
+import json
+import os
+import select
+import socket
+# import sys
 import rclpy
 from rclpy.node import Node
+from rcl_interfaces.msg import Log
+
+
+DEFAULT_UNIX_SOCKET_PATH = "/tmp/thi_drone"
+
+
+class MinimalPublisher(Node):
 
-from common_package_py.common_node import CommonNode
+    def __init__(self):
+        super().__init__('telemetry_node')
+        self.subscription = self.create_subscription(
+            Log,
+            '/rosout',
+            self.run_server, 
+            100)
+        self.subscription 
```

Comment:
What is this statement doing?
Looks like a simple void statement that could (should) be left out.

Reply:
The self.subscription "empty" statement was copied from the minimal subscriber provided by the ROS2 tutorials. It is used to avoid any unused variable warnings.

Reply:
![Screenshot 2024-05-06 224913](https://github.com/THI-Drone/telemetry_package/assets/166122914/16f677df-8dea-477e-8333-2ab0f735fa62)

Reply:
Have you observed the warning?
Otherwise I would just leave it if you cannot reproduce it.
At least put a comment next to it to clarify the purpose.